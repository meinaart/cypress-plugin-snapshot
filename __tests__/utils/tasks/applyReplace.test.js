/* eslint no-template-curly-in-string: 0 */
describe('utils/tasks/applyReplace', () => {
  const applyReplace = require('../../../src/utils/tasks/applyReplace');

  it('object replace', () => {
    const expected = {
      "text": "Hi ${name} & ${name}",
      "text2": "Hi ${name}",
    };
    const result = applyReplace(expected, {
      name: 'Meinaart',
    });
    expect(result).toMatchSnapshot();
  });
})
