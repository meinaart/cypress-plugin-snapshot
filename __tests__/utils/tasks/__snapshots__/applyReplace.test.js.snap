// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils/tasks/applyReplace object replace 1`] = `
Object {
  "text": "Hi Meinaart & Meinaart",
  "text2": "Hi Meinaart",
}
`;
